sanic/asgi.py

```diff
@@ -134,7 +134,7 @@ async def create(
 
         headers = Header(
             [
-                (key.decode("latin-1"), value.decode("latin-1"))
+                (key.decode(), value.decode())
```

Comment:
Add `errors="surrogateescape"` to match what we do in http1.py, this will handle invalid UTF-8 as well.

Reply:
For key, use `.decode("ASCII")` as per RFC 7230, which defines field-name as *token*, which can according to this RFC contain only visible US-ASCII characters (VCHAR) sans certain separator characters (or by other definition, certain punctuation and DIGIT and ALPHA, where ALPHA is not clearly defined but apparently only refers to ASCII letters).

For value, `.decode(errors="surrogateescape")` as stated, because they can contain opaque bytes other than UTF-8.

Reply:
For reference
https://www.rfc-editor.org/rfc/rfc7230#section-1.2  - VCHAR, ALPHA and DIGIT
https://www.rfc-editor.org/rfc/rfc7230#section-3.2.6  - token (field-name)
https://www.rfc-editor.org/rfc/rfc7230#section-3.2.4  - obs-text (in field-content)
https://www.rfc-editor.org/rfc/rfc7230#section-3.2  - field-name, field-content

Reply:
Added